tests/test_http.py

```diff
@@ -113,7 +116,45 @@ def request(self, *args, **kwargs):
       return httplib2.Response(self.success_json), self.success_data
     else:
       self.num_errors -= 1
-      raise ssl.SSLError()
+      if self.num_errors == 1:
+        raise ssl.SSLError()
+      else:
+        if sys.version_info[0] == 3:
```

Comment:
`six.PY3` is already imported into this module and is the general way that we determine twoness and threeness throughout this codebase. Any reason why this location should be different?